src/Classes/ServiceAPI/MyRadio_Swagger2.php

```diff
@@ -348,6 +370,44 @@ private static function getTags($apis)
         return $tags;
     }
 
+    private static function getMethodOpType($method)
+    {
+        $name = $method->getName();
+
+        //Note the ordering is important - create is static!
+        if (
+            CoreUtils::startsWith($name, 'create') ||
+            CoreUtils::startsWith($name, 'add')
+        ) {
```

Comment:
Don't think I'm a fan of this overly split style of if statements. Put it back on one line? I don't think it's that long anyway, and we've never overly cared about line length before...